src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -222,18 +222,26 @@ define([
       }
     },
 
-    destroyDeployment: function(deployment, component) {
+    destroyDeployment: function(deployment, component, force) {
       component.setLoading(true);
-      if (confirm("Destroy deployment of apps: '" + deployment.affectedAppsString() +
-          "'?\nDestroying this deployment will create and start a new deployment to revert the affected app to its previous version.")) {
 
+      var confirmMessage = !force ?
+        "Destroy deployment of apps: '" + deployment.affectedAppsString() +
+          "'?\nDestroying this deployment will create and start a new deployment to revert the affected app to its previous version." :
+        "Stop deployment of apps: '" + deployment.affectedAppsString() +
+          "'?\nThis will stop the deployment immediately and let it in the current state.";
```

Comment:
Maybe instead: "This will stop the deployment immediately and leave it in the current state."?